### PR TITLE
feat(cb2-7027): create light vehicle test group 3

### PR DESCRIPTION
**Pull request by tahayazici**
## Create a Light Vehicle Test Group 3

[CB2-7027](https://dvsa.atlassian.net/browse/CB2-7027)

### CODE REVIEW

File: src/app/forms/templates/test-records/section-templates/vehicle/group-3-light-vehicle-section.template.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    {
+      name: 'trailerId',
+      label: 'Trailer ID',
+      value: '',
+      disabled: true,
+      width: FormNodeWidth.L,
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
```
Comment (gjulien-bjss) [inline src/app/forms/templates/test-records/section-templates/vehicle/group-3-light-vehicle-section.template.ts:31]: Having the trailer id on here doesn't seem quite right

Comment (tahayazici) [reply src/app/forms/templates/test-records/section-templates/vehicle/group-3-light-vehicle-section.template.ts]: Removed.

**Review by Ian-Hodges — APPROVED** [review]

**Review by gjulien-bjss — DISMISSED** [review]

File: src/app/forms/templates/test-records/master.template.ts [hunk 2]
```diff
@@ -475,6 +476,15 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       notes: NotesSection,
       reasonForCreation: reasonForCreationSection
     },
+    testTypesDeskBasedGroup3: {
```
Comment (leifkemp-bjss) [inline src/app/forms/templates/test-records/master.template.ts:479]: This is in the master template, which is for amending tests.
Sections for creating tests go in the create master template.

Comment (leifkemp-bjss) [inline src/app/forms/templates/test-records/master.template.ts:479]: It's also in the LGV section for the master template, and this test group is for Small Trailers
I forget if we confirmed what we were going to do about light trailers or not, but I don't think it should be in this part? (correct me if I'm wrong)

Comment (tahayazici) [reply src/app/forms/templates/test-records/master.template.ts:479]: Thanks for pointing that out. I have not done any tests in a long while so I was not aware of the difference. I will address this.

Comment (tahayazici) [reply src/app/forms/templates/test-records/master.template.ts:479]: Good point man, after reading the ticket carefully, it looks like this PR is blocked. The Light TRL vehicle type needs to be implemented first for this to work. I will bring it in tomorrow's stand up.

Comment (tahayazici) [reply src/app/forms/templates/test-records/master.template.ts]: This now sits under `small trl`.

Comment (tahayazici) [reply src/app/forms/templates/test-records/master.template.ts]: This is now moved into the create master template.

**Review by gjulien-bjss — APPROVED** [review]

**Review by Ruairidh-BJSS — APPROVED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Pull request merged**